style.css

```diff
@@ -0,0 +1,178 @@
+* {
+  padding: 0;
+  margin: 2px;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 100vh;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.navbar {
+  justify-content: space-between;
+}
+
+nav {
+  background-color: white;
+  height: 80px;
+}
+
+nav h1.logo {
+  display: flex;
+  font-size: 18px;
+  line-height: 80px;
+  font-weight: bold;
+  color: #6070ff;
+  text-align: center;
+  align-items: center;
+  justify-content: flex-start;
+  margin: 2%;
+}
+
+nav ul {
+  float: right;
+  margin-right: 20px;
+}
+
+nav ul li {
+  display: inline-block;
+  line-height: 80px;
+  margin: 0 5px;
+}
+
+a:hover {
+  background: #acb7c3;
+  transition: 0.4s;
+}
+
+.nav-items {
+  margin: 16px 0;
+  transition: 0.7s ease;
+}
+
+.intro {
+  position: static;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  flex-direction: column;
+  height: 50vh;
+  justify-content: flex-start;
+  top: 30%;
+  left: 20%;
+  width: 62%;
+}
+
+.homepage {
+  display: flex;
+  justify-content: flex-start;
+  margin: 5% 10%;
+}
+
+section h1 {
+  display: flex;
+  position: static;
+  font-family: sans-serif;
+  color: #091e42;
+  font-size: 48px;
+  flex: none;
+  width: 80%;
+}
+
+section p {
+  display: flex;
+  position: static;
+  font-family: sans-serif;
+  color: #344563;
+  font-size: 20px;
+  line-height: 28px;
+  flex: none;
+  width: 80%;
+}
+
+.hamburger {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  align-self: center;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+@media (max-width: 950px) {
+  .hamburger {
+    display: block;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+}
+
+@media (max-width: 800px) {
+  .checkbtn {
+    display: block;
+  }
+}
+
+.nav-menu {
+  position: fixed;
+  left: 100%;
+  top: 70px;
+  gap: 0;
+  flex-direction: row;
+  background-color: #6070ff;
+  width: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.nav-menu.active {
+  left: 0;
+}
+
+.socials {
+  color: #7fbcff;
+  font-size: 16px;
+  padding-bottom: 20px;
+}
+
+.socials:hover {
+  background: #acb7c3;
+  transition: 0.4s;
+}
+
+.icon {
+  width: 20px;
+  margin: 2px;
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
- [ ] [OPTIONAL] Good job implementing a media query. However, this was not a requirement. I suggest you keep your scope around the tasks requirements.